GPU/D3D11/FramebufferManagerD3D11.cpp

```diff
@@ -824,12 +824,16 @@ void FramebufferManagerD3D11::PackFramebufferD3D11_(VirtualFramebuffer *vfb, int
 	const u32 fb_address = (0x04000000) | vfb->fb_address;
 	const int dstBpp = vfb->format == GE_FORMAT_8888 ? 4 : 2;
 
+	// TODO: Handle the other formats?  We don't currently create them, I think.
```

Comment:
I think this TODO may have been for the hardcoded `4` rather than for the dstBpp.

-[Unknown]